recipes/pyxtal/meta.yaml

```
@@ -0,0 +1,50 @@
+{% set name = "pyxtal" %}
+{% set version = "0.5.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pyxtal-{{ version }}.tar.gz
+  sha256: da6de3798bbed93047b2e8a30c05b36576219391a14225a3645e2ad5c82ee9fc
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  noarch: python
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
```

The linter should've caught these ones. Is 3.6 a good lower limit?

```suggestion
    - python >=3.6
    - pip
  run:
    - python >=3.6
```